application.conf

```diff
@@ -38,9 +39,12 @@ dm {
   analysesPath="/analyses"
   analysesResolvePath="/analyses/%s"
   analysesUpdatePath="/analyses/%s/outputs"
+  ubamCollectionPath="/collections"
```

Comment:
We're going to change the path to "/ubamcollections". We'll make sure everything is matching, I have some things to change in both vault-api and vault-datamanagement.